services/gateway/apis/http-version1/methods/voters.js

```diff
@@ -19,7 +19,7 @@ const { transformParams, response } = require('../swagger/utils');
 
 module.exports = {
 	version: '2.0',
-	swaggerApiPath: '/voters',
+	swaggerApiPath: '/votes_received',
 	rpcMethod: 'get.voters',
```

Comment:
The RPC method name should be updated as well. Can you please double-check `votes_sent` as well?

Reply:
Updated in both votes and voters.